src/TogglePassword/doc/index.rst

```diff
@@ -259,6 +259,35 @@ Then in your form, add your controller as an HTML attribute::
         // ...
     }
 
+Usage without Symfony Forms
+---------------------------
+
+You can also use the TogglePassword with native HTML inputs.
+It requires a little bit more work but it's still possible.
+Here is an example::
+
+    // ...
+
+    <label for="password">Password</label>
+    <input
+        id="password"
+        name="password"
+        type="password"
+        autocomplete="current-password"
+        required
+        placeholder="Password"
+        {{ stimulus_controller('@symfony/ux-toggle-password--toggle-password', {
+                visibleIcon: 'Default', // Here you can pass an SVG tag or a string. Default is the value for the default svg
+                visibleLabel: 'Show', // This field is optional
```

Comment:
```suggestion
                visibleLabel: 'Show', // optional
```

Reply:
I'll move it to `controller.ts` too

Reply:
@weaverryan I would like to propose this : 
```html+twig
            {{ stimulus_controller('symfony/ux-toggle-password/toggle-password', {
                    {# visibleLabel: 'Show password', // If you want to modify this label. #}
                    {# visibleIcon: 'Some svg icon', // If you want to modify this icon. #}
                    {# hiddenLabel: 'Hide password', // If you want to modify this label. #}
                    {# hiddenIcon: 'Some svg icon', // If you want to modify this icon. #}
                    buttonClasses: ['toggle-password-button'], // Add as many classes as you wish. "toggle-password-button" is needed to activate the default CSS.
            }) }}
```

WDYT ?

Reply:
Yes, that's a good idea, as these are different than the form field options (which are camelcase), so mentioning these here is a good idea 👍

Reply:
I have fixed this here : https://github.com/symfony/ux/pull/1125